about.html

```diff
@@ -12,60 +12,84 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+     <!-- navigation bar and logo -->
+     <nav>
```

Comment:
```suggestion
     <div class="navLinks>
```

Reply:
Consider switching the nav and the div. It's more common, thus more readable, to have nav elements as the parent to navigation links (a tags). Then you can wrap that navigation in a div if you'd like (for styling purposes).